R/step_population_scaling.R

```diff
@@ -0,0 +1,193 @@
+#' Create a recipe step that scales variables using population data
+#'
+#' `step_population_scaling` creates a specification of a recipe step
+#' that will add a population scaled column in the data. For example,
+#' load a dataset that contains county population, and join to an `epi_df`
+#' that currently only contains number of new cases by county. Once scaled,
+#' predictions can be made on case rate.
```

Comment:
I think we should mention that there's nothing special about "population" (just that that's the most natural/common use case). It could scale by any variable.